src/builder/ModuleHelper.js

```diff
@@ -130,7 +130,7 @@ class ModuleHelper {
     }
   }
 
-  async installModule(name) {
+  async installModule(name, installer) {
```

Comment:
I think the name `installer` is very confusing. what about `descriptor` ?

Reply:
I knew you will find a better name for it. I could not :)